lib/process-services/src/lib/process-list/components/process-filters.component.ts

```diff
@@ -71,15 +73,29 @@ export class ProcessFiltersComponent implements OnInit, OnChanges {
 
     filters: UserProcessInstanceFilterRepresentation [] = [];
     active = false;
+    currentRoute: string= '';
+    isProcessRoute: boolean;
+    private onDestroy$ = new Subject<boolean>();
 
     private iconsMDL: IconModel;
 
     constructor(private processFilterService: ProcessFilterService,
-                private appsProcessService: AppsProcessService) {
+                private appsProcessService: AppsProcessService,
+                private router: Router) {
     }
 
     ngOnInit() {
         this.iconsMDL = new IconModel();
+        this.router.events
+        .pipe(
+            filter((event) => event instanceof NavigationStart),
+            takeUntil(this.onDestroy$)
+        )
+        .subscribe((navigationStart: NavigationStart) => {
+            this.currentRoute = navigationStart.url;
```

Comment:
you probably don't need this variable, as component is not using "currentRoute" other than on the next line